src/app/core/components/marios-dialog/marios-dialog.component.ts

```diff
@@ -9,8 +9,10 @@ import { Employee } from '../../models/employee.model';
   styleUrls: ['./marios-dialog.component.scss'],
 })
 export class MariosDialogComponent {
+  @Input() isShowSender = true;
+
   constructor(
-    @Inject(MAT_DIALOG_DATA) public data: { marios: Marios; employee: Employee },
+    @Inject(MAT_DIALOG_DATA) public data: { marios: Marios; employee: Employee, isShowSender: boolean },
```

Comment:
Here you have isShowSender and in next file isSender, you can unify the name

Reply:
d881a003a9057976959a079fa595be4fb074d2a8